### PR TITLE
Enable replica validation by default

### PR DESCRIPTION
**Pull request by xinlian12**
Enable Replica validation by default

### CODE REVIEW

**Review by kushagraThapar — APPROVED** [review]

File: sdk/cosmos/azure-cosmos/CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,7 @@
 * Fixed a race condition that could result in a memory/thread leak for `BulkExecutor` instances (and their corresponding `cosmos-daemon-BulkExecutor-*` thread). - See [PR 31082](https://github.com/Azure/azure-sdk-for-java/pull/31082)
 
 #### Other Changes
+* Enable replica validation by default - See [PR 31159](https://github.com/Azure/azure-sdk-for-java/pull/31159)
```
Comment (kirankumarkolli) [inline sdk/cosmos/azure-cosmos/CHANGELOG.md:14]: May be worth adding some more context on this feature explicitly in the release notes.

**Review by kirankumarkolli — APPROVED** [review]

File: sdk/cosmos/azure-cosmos/CHANGELOG.md (hunk 1 above)
Comment (xinlian12) [reply sdk/cosmos/azure-cosmos/CHANGELOG.md:14]: @kirankumarkolli  any suggestions here?

**Comment by xinlian12** [comment]
Run the following tests locally and it passed:
partitionedSessionToken

File: sdk/cosmos/azure-cosmos/CHANGELOG.md (hunk 1 above)
Comment (xinlian12) [reply sdk/cosmos/azure-cosmos/CHANGELOG.md]: Removed from the changelog

**Pull request merged**